server/sonar-server/src/test/java/org/sonar/server/computation/task/projectanalysis/issue/IssueCreationDateCalculatorTest.java

```diff
@@ -70,6 +82,8 @@ public void before() {
     baseAnalysis = mock(Analysis.class);
     calculator = new IssueCreationDateCalculator(analysisMetadataHolder, scmInfoRepository, issueUpdater, activeRulesHolder);
 
+    when(activeRulesHolder.get(any(RuleKey.class)))
```

Comment:
No need to use a mock. The implementation `ActiveRulesHolderRule` has been written for tests.